docs/tutorial/flink.md

```diff
@@ -1,94 +1,407 @@
+       "root_path": "s3://<bucket-name>/<some-prefix>/{{job_name}}/output"
+     },
+     "InputCacheStorage": {
+       "fsspec_class": "s3fs.S3FileSystem",
+         "fsspec_args": {
+          "key": "<your-aws-access-key>",
+          "secret": "<your-aws-access-secret>",
+          "client_kwargs":{"region_name":"<your-aws-bucket-region>"}
+        },
+       // Input data cache should *not* be partitioned by `{{job_name}}`, as we want to get the datafile from the source only once
+       "root_path": "s3://<bucket-name>/<some-prefix>/input/cache"
+     }
+   }
+   ```
+
+2. `traitlet` configuration:
+
+   ```python
+   BUCKET_PREFIX = "s3://<bucket-name>/<some-prefix>/"
+   # The storage backend we want
+   s3_fsspec = "s3fs.S3FileSystem"
+   # Credentials for the backend
+   s3_args = {
+       "key": "<your-aws-access-key>",
+       "secret": "<your-aws-access-secret>",
+       "client_kwargs":{"region_name":"<your-aws-bucket-region>"}
+   }
+   # Take note: this is just python. We can reuse these values below
+
+   c.TargetStorage.fsspec_class = s3_fsspec
+   # Target output should be partitioned by `{{job_name}}`
+   c.TargetStorage.root_path = f"{BUCKET_PREFIX}/{{job_name}}/output"
+   c.TargetStorage.fsspec_args = s3_args
+
+   c.InputCacheStorage.fsspec_class = filesystem_class
+   c.InputCacheStorage.fsspec_args = s3_args
+   # Input data cache should *not* be partitioned by `{{job_name}}`, as we want to get the datafile from the source only once
+   c.InputCacheStorage.root_path = f"{BUCKET_PREFIX}/cache/input"
+   ```
+
+
+### Other Configuration Options
+
+A [subset of the configuration schema](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/main/pangeo_forge_recipes/injections.py) 
+that we just talked about related to inputs/outputs/caches gets dependency injected into the recipe by the runner. 
+
+Various other runner options ([documented here](https://pangeo-forge-runner.readthedocs.io/en/latest/reference/index.html) and [flink-specific here](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/)) 
+can also be passed either in the file configuration format above or directly during CLI `bake` calls. 
 
-You can add `--prune` if you want to only test the recipe and run just the first
-few steps.
+Here's a quick example of something slightly more complicated where you're passing [flink-specific configuration options](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/) and runner options. 
+Note that `-f <path-to-your-runner-config>.<json||py>` would point to our JSON or `traitlet` runner configuration file we just talked about above:
 
-This might take a minute to submit.
+   ```bash
+   pangeo-forge-runner bake \
+       --repo="https://github.com/pforgetest/gpcp-from-gcs-feedstock.git"  \
+       --ref="0.10.3" \
+       -f <path-to-your-runner-config>.<json||py> \
+       --FlinkOperatorBakery.job_manager_resources='{"memory": "2048m", "cpu": 1.0}' \
```

Comment:
IMO, a lot of these should be passed on to the runner via the config file, rather than as commandline parameters. Particularly putting inline JSON in commandline parameters is much harder to edit, and much easier to make mistakes in.

I would suggest that basically everything here except --repo, --ref and job_name should be in the config.

Reply:
yep, changed